arrows/pdal/algo/pointcloud_io.cxx

```diff
@@ -31,11 +31,11 @@ save_(vital::path_t const& filename,
       std::vector<vital::rgb_color> const& colors) const
 {
   namespace kv = kwiver::vital;
-  kv::logger_handle_t logger( kv::get_logger( "pdal_pointcloud_io_save_" ) );
+  kv::logger_handle_t logger( kv::get_logger( "pdal::pointcloud_io::save_" ) );
```

Comment:
I missed this before.  It should be "arrows.pdal.pointcloud_io".

It's not required, but typically we use `attach_logger("arrows.pdal.pointcloud_io")` in the constructor of this class and then access the logger latter with `logger()` as you did in the base class.